schemas/schema.js

```diff
@@ -127,21 +127,44 @@ const schema = {
               ],
             },
             {
-              name: "repositoryURL",
-              component: "text-field",
-              label: "Link to Github (or other) repository",
-              helperText: "Required for open source software - link to public repository",
+              component: "field-array",
+              name: "repositories",
+              label: "List of repositories",
+              description:
+                "Storage location for software packages typically managed by source control",
+              minItems: 1,
               condition: {
                 when: "type",
                 pattern: /software/,
               },
-              isRequired: true,
-              validate: [
+              fields: [
                 {
-                  type: "required",
+                  name: "name",
+                  component: "text-field",
+                  label: "Repository name",
+                  helperText: "For example: main, frontend, backend etc.",
+                  isRequired: true,
+                  validate: [
+                    {
+                      type: "required",
+                    },
+                  ],
                 },
                 {
-                  type: "url",
+                  name: "repoURL",
```

Comment:
As per your own changes to the candidates schema, this should be `url`, refer to [this line](https://github.com/DPGAlliance/publicgoods-candidates/blob/8ec299423ecc43c0bbff78967349ec191fac0ec6/nominee-schema.json#L349).

Reply:
Fixed in commit f88f06b